### ISSUE TITLE
Accessibility violations from Axe (CSV) does not support violation type "nested-interactive"

### ISSUE THREAD

**Issue by stevenwennekes** [ICTU/quality-time#3628]
Violations with type 'nested-interactive' do not show up as accessibility violation.
Sample row Axe CSV:
URL,Violation Type,Impact,Help,HTML Element,Messages,DOM Element
https://mywebsite.org,nested-interactive,serious,help,[div role='button'],Element has focusable descendants,.my-area
